src/back/TicketManager/tests/TicketManager.IntegrationTests/Events/UserInteractingWithEventTests.cs

```diff
@@ -111,6 +111,6 @@ private async Task VerifyEventsAsync(params EventDto[] events)
             PageSize = 10,
         });
 
-        actual.Items.Should().BeEquivalentTo(events);
+        actual.Items.Should().BeEquivalentTo(events, options => options.Excluding(e => e.Status));
```

Comment:
In this case you can probably very easily set right status to every event. There is no need to exclude that.